exercises/F-predicates/exercise.js

```diff
@@ -6,11 +6,15 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
```

Comment:
Hi Gulnihaln, 
Good work.
Just here I think you need to look again to your good .
function isNegative(number){
  if (number === 5){
    return false;
  } else {
      return true;
  }
}

Reply:
Hi Abed, Thanks for your review. I think I understood what you meant. If I'm right, when I set the function as in your example, it will only work for 5. 5 is  just a test number, it could be anything. Now, it works for any negative number.